src/langs/fr/auth.ts

```diff
@@ -0,0 +1,15 @@
+import { AUTH } from '../../constants/auth';
+
+export default {
+  [AUTH.SIGN_IN_BUTTON]: 'Se Connecter',
```

Comment:
For consistency, i think we should either capitalize all verbs, all capitalize none. In French, it probably makes more sense to not capitalize them